CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Zappa Changelog
 
+## 0.56.1
+
+* Use ZAPPA_RUNNING_IN_DOCKER environment variable to use any Python version inside Docker container (#1140)
```

Comment:
Add the following as well, as this resolves the issue there.

https://github.com/zappa/Zappa/issues/1201